### DISCUSSION TITLE
[Suggestion] Simpler alerts and confirmation modals

### DISCUSSION THREAD

**Discussion by lucasromanojf** [livewire/flux#133] [Feature Requests]
Hello,

currently alert and confirmation modals can be implemented by using the default modal component.

I'd like to suggest simpler modals for alerts which would have something like "title", "content" and "type" ("success", "warning", "error" and "info") props and maybe a slot for buttons (with an OK or Close button as default), and they could also be triggerable using Alpine (by using browser events).

The same idea would be followed for confirmation modals.

**Comment by joshhanley** [comment]
Thanks for reporting! We have converted this issue to a feature request for now, so we can focus on fixing any bugs first.

**Comment by git-alfacc** [comment]
No implementation yet?
It's useful when you need to have the user confirm checks.
